src/config/mod.rs

```diff
@@ -17,6 +17,21 @@ static INSTANCE: OnceCell<Config> = OnceCell::new();
 
 const CONFIG_FILE_NAME: &str = "cli.yml";
 
+// This is the default environment name.
+pub const DEFAULT_ENV_NAME: &str = "default";
+
+// Default prefix for environment variables added by CloudTruth.
+pub const ENV_VAR_PREFIX: &str = "CT_";
+
+// Environment variable name used to specify the CloudTruth API value, so it does not need to be
+// specified on the command line.
+#[allow(dead_code)]
```

Comment:
We should try to keep dead code to a minimum. The only place it exists now is in the GraphQL operations and that was done to represent all our potential resources and actions so that code didn't inadvertently get split up over multiple places. This definition is a bit funky because it isn't a declaration so much as a derived value, joined from the `ENV_VAR_PREFIX` and the config keys.

If it's only used by tests, we could put it in the test module as well.

Reply:
I was trying to have a centralized location for the CT environment variable name definitions. I did change the main code block so it is obvious where those are used, instead of a generating the value inside `get_override()`.

Reply:
Rick and I chatted about this separately. Just to close the loop on it, we used to use [config-rs](https://github.com/mehcode/config-rs) to manage the environment name overrides of config file values. We moved away from that when we added support for multiple profiles. The library didn't handle that in a straightforward manner and since we decided to only support YAML configuration, it became easier just handling the config code ourselves. Some of the dynamic key naming was a direct result of the _config-rs_ options. Without that in place, we no longer need to dynamically look up values that way and thus the simplifications Rick is making to `get_override()`.